_layouts/styleguide.html

```diff
@@ -17,7 +17,9 @@
       aria-label="On this page"
       data-scroll-offset="-120"
       data-root-margin="48px 0px -90% 0px"
-      data-threshold="1"></aside>
+      data-threshold="1"
+      data-heading-elements={{ page.in_page_nav }}
```

Comment:
I considered adding a check here for certain values, but ultimately moved away from this because it felt unnecessarily constraining. Let me know if I should add something like this in!
```suggestion
      {% if page.in_page_nav == "h2" or page.in_page_nav == "h3" %}
          data-heading-elements={{ page.in_page_nav }}
      {% endif %}
```

Reply:
What would this change do?

Reply:
The update shown in the suggested change would just prevent the attribute from being added unless the variable receives one of the expected values.

The benefit of this is that the rendered code would be cleaner (the attribute wouldn't be added with an empty value). 

The downside of this is that it either limits the possible values we can enter OR we'd have to check for all possible combinations here. For example, maybe we will have a need to show `h3` and `h4` values in the in-page nav on a given page, or maybe we would want `h2`, `h3`, and `h4`. 

I opted for an open value for the attribute here for maximum flexibility.